### PR TITLE
Use image.IsUnpacked

### LINKED ISSUE #361: Use `image.IsUnpacked` in containerd.
In https://github.com/kubernetes-incubator/cri-containerd/pull/358, we need to check whether image is unpacked before container creation.

We could use the new helper function in containerd. https://github.com/containerd/containerd/pull/1668

### PR DESCRIPTION
**Pull request by yanxuean**
fix #361
Signed-off-by: yanxuean <yan.xuean@zte.com.cn>

### CODE REVIEW

File: pkg/server/restart.go [hunk 1]
```diff
@@ -355,6 +354,11 @@ func loadImages(ctx context.Context, cImages []containerd.Image, provider conten
 			glog.Warningf("The image content readiness for %q is not ok", i.Name())
 			continue
 		}
+		// Checking existence of top-level snapshot for each image being recovered.
+		if unpacked, err := i.IsUnpacked(ctx, snapshotter); err != nil || !unpacked {
```
Comment (Random-Liu) [inline pkg/server/restart.go]: Let's distinguish error and `unpacked`.

Comment (yanxuean) [reply pkg/server/restart.go]: done

**Pull request merged**